ambry-coordinator/src/test/java/com.github.ambry.coordinator/MockBlockingChannel.java

```diff
@@ -152,13 +156,14 @@ public void send(Send send)
             break;
 
           case Blob:
-            MockDataNode.BlobOutputAndError bdae = mockDataNode.getData(blobId);
+            MockDataNode.BlobDataAndError bdae = mockDataNode.getData(blobId);
             getResponseErrorCode = bdae.getError();
             if (getResponseErrorCode == ServerErrorCode.No_Error) {
-              BlobOutput blobData = bdae.getBlobOutput();
-              byteBufferSize = (int) MessageFormatRecord.Blob_Format_V1.getBlobRecordSize(blobData.getSize());
+              BlobData blobData = bdae.getBlobData();
+              byteBufferSize = (int) MessageFormatRecord.Blob_Format_V2.getBlobRecordSize(blobData.getSize());
               byteBuffer = ByteBuffer.allocate(byteBufferSize);
-              MessageFormatRecord.Blob_Format_V1.serializePartialBlobRecord(byteBuffer, blobData.getSize());
+              MessageFormatRecord.Blob_Format_V2
```

Comment:
Production code for puts in the coordinator is hardcoded today to use `DataBlob`. 

Production code deals with `BlobOutput` today.

Where do you think we have to add comments about legacy code and all? This is a test class. Let us not exaggerate its importance. Coordinator will never deal with metadata blobs in production, why go out of our way to have the tests deal with them?